### PR TITLE
feat: Added Navbar for mobile(including hamburger menu)

### PR DESCRIPTION
**Pull request by BalajiHarinath**
I have added nav bar for mobile screen. Please review.

Files to be reviewed:-
1. docs.html(only check the navigation part)
2. Components/navigation.cs
navbar-mobile

### CODE REVIEW

File: Components/navigation.css [hunk 1]
```diff
@@ -56,12 +69,43 @@
...
+    border: 1px solid var(--nav-violet);
+    background: var(--nav-white);
+    width: fit-content;
+}
+
+.icon-sidebar-nav:hover {
+    color: var(--nav-violet);
+}
+
+@media screen and (max-width: 666px) {
```
Comment (ShekharDhangar) [inline Components/navigation.css:107]: try to use media queries in ems

**Review by ShekharDhangar — APPROVED** [review]
Code is good.  But a live link would be good to test.

File: Components/navigation.css [hunk 2]
```diff
@@ -56,12 +69,43 @@
...
+    border: transparent;
+    cursor: pointer;
+}
+
+.container-sidebar-navigation {
+    display: flex;
+    flex-direction: column;
+    padding: 0rem 1rem 1rem 1rem;
+    border: 1px solid var(--nav-violet);
+    background: var(--nav-white);
```
Comment (anik31) [inline Components/navigation.css:99]: appreciation - good that you're using css variables & shorthand for padding

**Review by anik31 — APPROVED** [review]
The navbar & hamburger menu is looking good. Code is also clean

**Pull request merged**